docs/build/active-learning/index.md

```diff
@@ -0,0 +1,125 @@
+---
+icon: fontawesome/solid/wand-magic-sparkles
+subtitle: Use AI/ML to learn linking rules
+status: new
+tags:
+  - BeginnersTutorial
+---
+
+# Active Learning of Linking Rules
+
+## Introduction
+
+Active learning infuses expert knowledge and creates new relationships between properties of two datasets. We can learn new rules and refine existing rules.
+
+The documentation consists of the following parts described in detail below:
+
+-   [Introduction](#introduction)
+-   [Usage](#usage)
+-   [Start the learning dialog](#start-the-learning-dialog)
+-   [Creating an automatic link rule](#creating-an-automatic-link-rule)
+-   [Add property paths for both entities](#add-property-paths-for-both-entities)
+
+## Usage
+
+Define properties to compare between entities of the selected datasets.
+
+## Start the learning dialog
+
+Create a linking rule and setup source and target datasets as well as the linking property that should be yielded.
+Start the learning dialog by clicking the “Learning” tab in the linking view.
+
+## Creating an automatic link rule
+
+-   Choose properties to compare.
+    Select from the suggestions or search them by specifying property paths for both entities.
+
+    ![image](22.2-Suggestion.png){ class="bordered" }
```

Comment:
FYI - I introduced a new class in `extra.css` to be reused where needed.

Reply:
thank you 🙌